gooddata-sdk/gooddata_sdk/utils.py

```diff
@@ -306,3 +307,17 @@ def safeget_list(var: Any, path: List[str]) -> List[Any]:
         return []
     else:
         raise Exception("safeget_list: result is not iterable! result={0}".format(result))
+
+
+def get_namespace_from_xml(xliff_content: str):
+    """Extract the XML namespace from the given XLIFF content.
+
+    Args:
+        xliff_content (str): The XLIFF content as a string.
+
+    Returns:
+        dict: A dictionary containing the namespace with the key 'ns'.
+    """
+    tree = ET.ElementTree(ET.fromstring(xliff_content))
+    root = tree.getroot()
+    return {"ns": root.tag.split("}")[0].strip("{")}
```

Comment:
Fix the naming, please. The name of the method is `get_namespace_from_xml`, but the parameter is `xliff_content`.